sdks/python/apache_beam/ml/inference/sklearn_inference.py

```diff
@@ -18,7 +18,7 @@
 import enum
 import pickle
 import sys
-from typing import Any
+from typing import Any, Callable
 from typing import Dict
```

Comment:
Let's import `Callable` in a separate line
```suggestion
from typing import Any
form typing import Callable
from typing import Dict
```

Reply:
Done.